services/api/service.go

```diff
@@ -1777,6 +1780,410 @@ func (api *RelayAPI) handleGetPayload(w http.ResponseWriter, req *http.Request)
 	log.Info("execution payload delivered")
 }
 
+func (api *RelayAPI) handleGetPayloadV2(w http.ResponseWriter, req *http.Request) {
```

Comment:
Hmm can we deduplicate this function like we did in mev-boost? Maybe with a `innerHandleGetPayload` like mev-boost? After removing support for V1, we can simplify again.